crown-court-proceeding/src/main/java/uk/gov/justice/laa/crime/crowncourt/service/ProceedingService.java

```diff
@@ -69,6 +69,30 @@ public ApiUpdateApplicationResponse updateApplication(CrownCourtDTO dto) {
         return apiUpdateApplicationResponse;
     }
 
+
+    public Optional<Void> checkCCDetails(CrownCourtDTO dto) {
+        if (dto.getCrownCourtSummary() != null && dto.getCrownCourtSummary().getCrownCourtOutcome() != null
```

Comment:
If we extract out `dto.getCrownCourtSummary()` as a local variable outside of the if statements it will greatly simplify this block of code and aid readability